test-harness/specs/validate-body-params/json-invalid-request-2.pc.txt

```diff
@@ -48,4 +48,4 @@ curl -i -X POST http://localhost:4010/path -H "Content-Type: application/json" -
 HTTP/1.1 422 Unprocessable Entity
 content-type: application/problem+json
 
-{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request","status":422,"detail":"Your request is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["body"],"severity":"Error","code":"required","message":"must have required property 'status'"},{"location":["body","id"],"severity":"Error","code":"type","message":"must be number"}]}
+{"type":"https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY","title":"Invalid request","status":422,"detail":"Your request is not valid and no HTTP validation response was found in the spec, so Prism is generating this error for you.","validation":[{"location":["body","id"],"severity":"Error","code":"type","message":"must be integer"}]}
```

Comment:
same as above, since we are no longer saying ever property is required, updated the expected error message to reflect that.